WpfApp2/MainWindow.xaml.cs

```diff
@@ -54,15 +54,11 @@ public partial class MainWindow : Window {
             }
         }
 
-        private void ChangeFileSettingsStatus(string str)
-        {
-            if (String.IsNullOrEmpty(str)) {
-                FileSettingsStatus.Text = "";
-                return;
-            }
-            else {
-                FileSettingsStatus.Text = str;
-            }
+        private void ChangeStatuses(string statusBarText, bool buttonStatus) {
+            FileSettingsStatus.Text = statusBarText;
+            LoadButton.IsEnabled = buttonStatus;
```

Comment:
If you could, add a blank row to distinguish two separate events.
a) set status bar
b) set button status
Help readability. Cosmetics.

Reply:
Will do it tomorrow!